package.json

```diff
@@ -25,38 +25,33 @@
     "watch": "textlint-scripts build --watch",
     "prepublish": "npm run --if-present build",
     "test": "textlint-scripts test",
-    "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,css}\""
+    "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,css}\"",
+    "prepare": "git config --local core.hooksPath .githook"
   },
   "keywords": [
     "textlint",
     "textlintrule"
   ],
   "description": "ら抜き言葉を検出するtextlint rule",
   "devDependencies": {
-    "husky": "^4.3.0",
-    "lint-staged": "^10.4.2",
+    "lint-staged": "^10.5.4",
     "power-assert": "^1.4.1",
```

Comment:
Probably, we can remove `power-assert` deps.

Reply:
Thank you for reviewing.
I'll remove it.